client/my-sites/plans/ecommerce-trial/upgrade-confirmation/index.tsx

```diff
@@ -42,6 +43,16 @@ const TrialUpgradeConfirmation = () => {
 	} );
 
 	const currentPlanName = isFetchingSitePlan ? '' : selectedSite?.plan?.product_name_short ?? '';
+	const isEnrepreneurTrialFlow = config.isEnabled( 'entrepreneur-my-home' );
+	const entrepreneurTrialWelcome =
+		currentPlanName &&
+		translate( 'Welcome to the %(planName)s plan', {
+			args: { planName: currentPlanName },
+			comment: 'The `planName` is the name of the plan the user has just upgraded to.',
+		} );
```

Comment:
You can add `isEntrepreneurTrialFlow` as an additional condition for the translation to be performed. Otherwise, the translation will occur regardless of whether it is displayed or not.

Reply:
Good point. Thanks!

Since this code will have a short life - as we will be replacing the current logic once moved from Woo to Entrepreneur plan completely, I think we can keep the things as they are for now.